app/Actions/Import/Extensions/ImportPhoto.php

```diff
@@ -17,7 +17,7 @@ trait ImportPhoto
 	 *
 	 * @return bool returns true when photo import was successful
 	 */
-	public function photo($path, $delete_imported, $albumID = 0, $force_skip_duplicates = false, $resync_metadata = false)
+	public function photo($path, $delete_imported, $albumID = 0, $force_skip_duplicates = false, $import_via_symlink, $resync_metadata = false)
```

Comment:
I'm hardly an expert on PHP but shouldn't a non-optional argument (`$import_via_symlink`) be before the optional ones?

Reply:
I don't know enough PHP and was afraid that changing the order of the arguments would result in unexpected regressions, so I left it...